core/input.go

```diff
@@ -21,9 +21,8 @@ import (
 	"net/url"
 	"time"
 
+	"github.com/rond-authz/rond/logging"
```

Comment:
why logging and not logger? Why not an internal package?

Reply:
Yes, but the child package logrus must be used if you use logrus. Also, the test logger could be useful to test, maybe? idk
And the interface must be public

Reply:
makes sense, ok